src/no/runsafe/UserControl/database/PlayerDatabase.java

```diff
@@ -75,8 +75,11 @@ public ISchemaUpdate getSchemaUpdateQueries()
 			"ALTER TABLE player_db ADD COLUMN ban_by_uuid VARCHAR(36) NULL;",
 			// Convert banningPlayer usernames to UUIDs
 			"UPDATE IGNORE `player_db` SET `ban_by_uuid` = " +
-				"(SELECT `uuid` FROM `player_db` WHERE `name`=`player_db`.`ban_by`) " +
-				"WHERE `ban_by` IS NOT NULL;",
+				"(SELECT `uuid` FROM `player_username_log` WHERE `name`=`player_db`.`ban_by`) " +
+				"WHERE `ban_by` IS NOT NULL AND `ban_by` != 'console';",
+			// Convert banning console to UUID
+			"UPDATE IGNORE `player_db` SET `ban_by_uuid` = '" + playerUsernameLog.consoleUUID + "' " +
+				"WHERE `ban_by` == 'console';",

```

Comment:
If you change the order here, you can remove the check for ban_by != console above :)